src/utils/migrationUtils.ts

```diff
@@ -152,15 +152,17 @@ export const loadModule = async (migrationFile: string): Promise<MigrationModule
         });
 };
 
-export const loadMigrationFiles = async (): Promise<IMigration[]> => {
+export const loadMigrationFiles = async (range: [number, number] | null): Promise<IMigration[]> => {
     const migrations: IMigration[] = [];
 
     const files = listFiles('.js');
 
     for (const file of files) {
         const migrationModule = await loadModule(file.name);
 
```

Comment:
I might add conditions to check whether the range is within `migrationrange`.

Reply:
I didn't add the check as the current implementation is able to handle it too. If there is no migration that matches the specified range and at the same time is not in status.json, the run will end with the message: 'No migrations to run.'